spec/system/visit_msa_component_new_spec.rb

```diff
@@ -40,6 +41,22 @@
     end
   end
 
+  context 'creation fails without unique entity id' do
+    it 'when entity id already exists for a msa component' do
+      existing_component = msa_component
+      entity = 'existing entity_id'
```

Comment:
I believe entity IDs shouldn't be allowed to have spaces? This should also be validated (not necessarily part of this story but maybe worth fixing whilst we're at it)